model/cloud.go

```diff
@@ -218,3 +225,55 @@ type ProductLimits struct {
 	Messages     *MessagesLimits     `json:"messages,omitempty"`
 	Teams        *TeamsLimits        `json:"teams,omitempty"`
 }
+
+type NotifyAdminToUpgradeRequest struct {
+	CurrentTeamId string `json:"current_team_id"`
+}
+
+type UserInfo struct {
+	UserID    string
+	TimeStamp int64
```

Comment:
Nit: `Timestamp` instead of `TimeStamp`. It is a a single word, and most code in this codebase uses it as a single word.